src/vm.sh

```diff
@@ -109,14 +109,58 @@ function vm_up()
   cmd_manager "$flag" "$cmd"
 }
 
+function parse_vm_options()
+{
+  local long_options='help,mount,umount,up'
+  local short_options='h,mo,um,u'
+
+  options="$(getopt \
+    --name 'kw vm' \
+    --options "$short_options" \
+    --longoptions "$long_options" \
+    -- "$@")"
+
+  if [[ "$?" != 0 ]]; then
+    return 22 # EINVAL
+  fi
+
+  eval "set -- $options"
+
+  while [[ "$#" -gt 0 ]]; do
+    case "$1" in
+      --help | -h)
+        vm_help "$1"
+        exit
+        ;;
+      --mount | mo)
+        vm_mount "$@"
+        exit
+        ;;
+      --umount | um)
+        vm_umount "$@"
+        exit
+        ;;
+      --up | u)
+        vm_up "$@"
```

Comment:
For this part, I recommend that you create an associative array named `options_values` on top of this file (e.g., `declare -gA options_values`) with these entries:
1. `MOUNT`: Set 0 by default at the beginning of the parse function. Set 1 if we match this case.
2. `UMOUNT`: Set 0 by default and 1 if the user requests it.
3. `UP`: Same idea.

After that, you can create an "entry point" for this feature. For example, let's call it `vm_main`, and this function will call the parse function first, and next it will check the option. Something like this:

```bash
function vm_main()
{
  if [[ ... --help or -h .. ]]; then
   vm_help
   exit
  fi

  parse_vm_options "$@"
  if [[ "$?" != 0 ]]; then
   error...
  fi

  if [[ ${options_values['MOUNT']} == 1 ]]; then
    vm_mount "PARAMETERS"
  fi
  ...
}
```